librz/include/rz_util/rz_rbtree.h

```diff
@@ -55,6 +56,8 @@ typedef struct rz_containing_rb_tree_t {
 	RContRBFree free;
 } RContRBTree;
 
+typedef RBTree RBItvTree;
```

Comment:
Why not use `RzIntervalTree`?

Reply:
Good point, it will also remove the need for that additional code in `rbtree.c`